src/ManifoldsBase.jl

```diff
@@ -163,19 +163,23 @@ not approximately equal, containting a more detailed description/reason.
 If the two elements are approximalely equal, this method returns `nothing`.
 
 This method is an internal function and is called by `isapprox` whenever the user specifies
-an `error=` keyword therein.
+an `error=` keyword therein. `_isapprox` is another related internal function. It is
```

Comment:
Should we maybe add a doctoring to `_isapprox` as well?

Reply:
I've added a docstring there.